src/commonMain/kotlin/fr/acinq/lightning/channel/states/Syncing.kt

```diff
@@ -270,9 +270,12 @@ data class Syncing(val state: PersistedChannelState) : ChannelState() {
                     }
                     else -> unhandled(cmd)
                 }
+                val sendChannelReestablish = when (state) {
```

Comment:
We could add this boolean in the `Syncing` state, which is not persisted. It would save us from keeping consistency between `Offline` and `Syncing`.

Reply:
Good idea, done in https://github.com/ACINQ/lightning-kmp/pull/473/commits/80045cda2602f7d28100f95b430dee46b013decd